src/plays/drumkit/styles.css

```diff
@@ -0,0 +1,80 @@
+.play-details-body {
+  text-align: center;
+  background-color: #283149;
+}
+
+.heading {
+  font-size: 5rem;
+  color: #dbedf3;
+  font-family: 'Arvo', cursive;
+  text-shadow: 3px 0 #da0463;
+}
+
+.footer {
+  color: #dbedf3;
+  font-family: sans-serif;
+}

```

Comment:
hello @aaqib605 you have to rename classes to different names for your project, otherwise direct target can break the site. also there are lots of things to change so if you don't mind could you please create a new PR?